types/ibc.go

```diff
@@ -80,8 +80,12 @@ type IBCBasicResult struct {
 	Err string            `json:"error,omitempty"`
 }
 
-// IBCBasicResponse defines the return value on a successful processing
+// IBCBasicResponse defines the return value on a successful processing.
+// This is the counterpart of [IbcBasicResponse](https://github.com/CosmWasm/cosmwasm/blob/v0.14.0-beta1/packages/std/src/ibc.rs#L194-L216).
 type IBCBasicResponse struct {
+	// Submessages are like Messages, but they guarantee a reply to the calling contract
+	// after their execution, and return both success and error rather than auto-failing on error
+	Submessages []SubMsg `json:"submessages"`
```

Comment:
You also need this in the IbcReceiveResponse (at least I hope I added it there in Rust as well)

Reply:
Done in 46d8a32aa99827a31eb2b4763f0f9dbf933d3945